metadata-ingestion/src/datahub/utilities/sqlglot_lineage.py

```diff
@@ -1156,3 +1156,19 @@ def create_lineage_sql_parsed_result(
     finally:
         if needs_close:
             schema_resolver.close()
+
+
+def view_definition_lineage_helper(
+    result: SqlParsingResult, view_urn: str
+) -> SqlParsingResult:
+    if result.query_type is QueryType.SELECT:
+        # Some platforms (e.g. postgres) store only <select statement> from view definition
+        # `create view V as <select statement>` . For such view definitions, `result.out_tables` and
+        # `result.column_lineage[].downstream` are empty in `sqlglot_lineage` response, whereas
```

Comment:
looks like the comment got cut off here - not a huge thing, and we can fix in a follow up

Reply:
updated.